data/tvshows.js

```diff
@@ -1,58 +1,116 @@
 const axios = require("axios");
 const validation = require('./validation');
 
-async function searchTVShowsAPI(searchTerm) {
-    const {
-        data
-    } = await axios.get('http://api.tvmaze.com/search/shows?q=' + searchTerm);
-    return data;
-}
-
-async function getTVShowAPI(showID) {
-    const {
-        data
-    } = await axios.get('http://api.tvmaze.com/shows/' + showID);
+async function searchMovieAPI(searchTerm){
+    const {data} = await axios.get('https://www.omdbapi.com/?apikey=58db0176&s=' + searchTerm);
     return data;
 }
 
-async function searchTVShows(searchTerm) {
+async function searchMovie(searchTerm) {
 
     validation.validateString("searchTerm", searchTerm);
 
-    const data = await searchTVShowsAPI(searchTerm);
+    const data = await searchMovieAPI(searchTerm);
+    console.log("option a" + data.Search[0].imdbID);
```

Comment:
Remove loggers?

Reply:
yeah because we don't need searchtvshows or gettvshows anymore